src/main/java/com/exasol/adapter/dialects/redshift/RedshiftSqlDialect.java

```diff
@@ -112,6 +113,12 @@ public NullSorting getDefaultNullSorting() {
         return NullSorting.NULLS_SORTED_AT_END;
     }
 
+    @Override
+    // https://docs.aws.amazon.com/redshift/latest/dg/r_Examples_with_character_types.html
+    public String getStringLiteral(final String value) {
+        return super.quoteLiteralStringWithSingleQuote(value);
```

Comment:
The documentation says nothing about `''`. Probably have to test `''` and `\` manually.

Reply:
Here I found something: https://github.com/awsdocs/amazon-redshift-developer-guide/blob/master/doc_source/copy-parameters-data-conversion.md
Probably it's \

Reply:
I think that page is only about the COPY command.